app/models/question.rb

```diff
@@ -3,11 +3,15 @@ class Question < ActiveRecord::Base
   has_many :tags, as: :subscriber
   has_many :answers
   belongs_to :user
-
   validates :content, presence: true
 
   include Modify
 
+  def self.with_answers
```

Comment:
I think this could be in the `class << self` scope below, since it's a class method as well.